hack/jenkins/kicbase_auto_build.sh

```diff
@@ -126,5 +126,8 @@ else
 	git remote add minikube-bot git@github.com:minikube-bot/minikube.git
 	git push -f minikube-bot ${branch}
 
-	gh pr create --fill --base master --head minikube-bot:${branch}
+	pr=$(gh pr create --fill --base master --head minikube-bot:${branch} \
+	      | grep -o -E "github.com/kubernetes/minikube/pull/[0-9]+" \
+	      | grep -o -E '[0-9]+')
+	gh pr comment "$pr" --body "/ok-to-test"
```

Comment:
is there a way we could add the /ok-to-test only if the label is not already set ?

I think if the label is already set, the Image Push commit will start a job, and this comment will start another one, and it will make two jobs running per commit wdyt ?

Reply:
gh doesn't have a great of checking labels on a specific PR unfortunately

Reply:
we just created this PR, there is no way `/ok-to-test` is set there :)